tokens/src/components/DataTable.json

```diff
@@ -2,47 +2,43 @@
   "color": {
     "data-table": {
       "bg": {
-        "base": { "value": "{color.bg.base}", "source": "$data-table-background-color" },
+        "base": { "value": "{color.bg.base}", "type": "color", "source": "$data-table-background-color" },
         "is-loading": {
           "value": "{color.white}",
+          "type": "color",
           "source": "$data-table-is-loading-bg",
-          "modify": [
-            {
-              "type": "alpha",
-              "amount": 0.7
-            }
-          ]
+          "modify": [{ "type": "alpha", "amount": 0.7 }]
         }
       },
-      "border": { "value": "{color.gray.200}", "source": "$data-table-border-color" }
+      "border": { "value": "{color.gray.200}", "type": "color", "source": "$data-table-border-color" }
     }
   },
   "elevation": {
     "data-table": {
-      "box-shadow": { "value": "{elevation.box-shadow.sm}", "source": "$data-table-box-shadow" }
+      "box-shadow": { "value": "{elevation.box-shadow.sm}", "source": "$data-table-box-shadow", "type": "shadow" }
     }
   },
   "size": {
     "data-table": {
-      "border": { "value": "1px", "source": "$data-table-border" },
+      "border": { "value": "1px", "type": "dimension", "source": "$data-table-border" },
       "dropdown": {
         "pagination": {
-          "max-height": { "value": "60vh", "source": "$data-table-pagination-dropdown-max-height" },
-          "min-width": { "value": "6rem", "source": "$data-table-pagination-dropdown-min-width" }
+          "max-height": { "value": "60vh", "type": "dimension", "source": "$data-table-pagination-dropdown-max-height" },
+          "min-width": { "value": "6rem", "type": "dimension", "source": "$data-table-pagination-dropdown-min-width" }
         }
       }
     }
   },
   "spacing": {
     "data-table": {
       "padding": {
-        "x": { "value": ".75rem", "source": "$data-table-padding-x" },
-        "y": { "value": ".75rem", "source": "$data-table-padding-y" },
-        "small": { "value": ".5rem", "source": "$data-table-padding-small" },
-        "cell": { "value": ".75rem", "source": "$data-table-cell-padding" },
-        "head-cell": { "value": ".5rem .75rem", "source": "$data-table-head-cell-padding" }
+        "x": { "value": ".75rem", "type": "dimension", "source": "$data-table-padding-x" },
+        "y": { "value": ".75rem", "type": "dimension", "source": "$data-table-padding-y" },
+        "small": { "value": ".5rem", "type": "dimension", "source": "$data-table-padding-small" },
+        "cell": { "value": ".75rem", "type": "dimension", "source": "$data-table-cell-padding" },
+        "head-cell": { "value": ".5rem .75rem", "type": "dimension", "source": "$data-table-head-cell-padding" }
       },
-      "footer-position": { "value": "center", "source": "$data-table-footer-position" }
+      "footer-position": { "value": "center", "type": "ratio", "source": "$data-table-footer-position" }
```

Comment:
[curious] what is the `ratio` type representing here?

Reply:
@viktorrusakov proposed to use the `type: "position"`. It`s a good type for the CSS property `align-self` (`start`, `center`, `end`).

Reply:
yeah, I don't think W3C specs have a type for such properties, `position` sounds good to me.
 
@adamstankiewicz what do you think?

Reply:
`position` sounds good here, too +1